index.bs

```diff
@@ -676,6 +681,27 @@ Issue: Parse |value|["`filter_data`"].
 
 Issue: Parse |value|["`aggregation_keys`"].
 
+<h3 dfn id="should-block-processing-for-destination-limit">Should processing be blocked by destination limit</h3>
+
+Given an [=attribution source=] |source|:
```

Comment:
I think this should be written as
```
To <dfn>...</dfn> given...
```

Reply:
Thanks Andrew. Yeah, I agree, but I can't think of a good way to describe like this... Any suggestions?

Reply:
```
To <dfn>check if an attribution source meets the pending destination limit</dfn>...
```

Something like that?